### PR TITLE
Add first edu edition elements and edit enchantments

### PR DESCRIPTION
**Pull request by Kaooot**

### CODE REVIEW

File: gomint-api/src/main/java/io/gomint/world/block/BlockBeryllium.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package io.gomint.world.block;
+
+/**
+ * @author Kaooot
+ * @version 1.0
+ */
+public interface BlockBeryllium extends Block {
```
Comment (Janmm14) [inline gomint-api/src/main/java/io/gomint/world/block/BlockBeryllium.java]: extends api BlockElement, same for other blocks in api

**Review by Janmm14 — CHANGES_REQUESTED** [review]

**Comment by geNAZt** [comment]
Elements have multiple neutron values. The PR in its current state does not reflect that data.

**Comment by Janmm14** [comment]
Weight is dependant on neutrons. How should that be handled?

**Comment by Kaooot** [comment]
Only the element Fluorine has two values given on the atomic weight. I think, I will use a int[] for the atomic weight as return.

File: gomint-api/src/main/java/io/gomint/world/block/BlockElement.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * @author Kaooot
+ * @version 1.0
+ */
+public interface BlockElement extends Block {
+
+    int getProtons();
+
+    int getElectrons();
+
+    int[] getNeutrons();
```
Comment (geNAZt) [inline gomint-api/src/main/java/io/gomint/world/block/BlockElement.java:13]: Please add a documentation why a element has multiple neutron values and what todo with it

**Review by geNAZt — CHANGES_REQUESTED** [review]
In general how does MC:PE handle the different neutron settings in Items/Blocks? There needs to be a way that this data isstored somewhere?

**Comment by lukeeey** [comment]
Is this still be worked on @Kaooot ?

**Comment by Kaooot** [comment]
nope you can close this pr if you want

**Comment by XDqrkneZ** [comment]
Why am I reading science words here

**Pull request closed without merging**